src/asset/unit.ts

```diff
@@ -0,0 +1,23 @@
+import {Asset, Wallet} from "../interfaces";
+import ApiPromise from "@polkadot/api/promise";
+import {getBalance} from "../rpc/substrate/getBalance";
+import {getAddress} from "../rpc/getAddress";
+import {executeAssetOperation} from "./executeAssetOperation";
+
+export function unitAsset(balance: string, address: string): Asset {
+  return {
+    balance: balance,
+    customViewUrl: `https://polkascan.io/pre/kusama/account/${address}`,
+    decimals: 0,
+    identifier: 'unit-test-asset',
+    image: '',
```

Comment:
put this logo:
https://coinranking.com/coin/ePlOuwd_e+kusama-ksm

Reply:
@mpetrunic I am not sure why the image is not showing. I checked their [examples repos](https://github.com/MetaMask/snaps-cli/tree/master/examples/custom-token) and don't see any problem with our image. I even scaled it to 200x200 (as in the example) and rehosted it but it won't show. I will leave it for now (left URL inside code but it is not working)

Reply:
we'll figure it out